File: Iniciante/1048 - Aumento de Salário.py
"""
Autor: Pierre Vieira
Data da submissão: 25/01/2019 21:16:06
"""
salario = float(input())
if 0 <= salario <= 400: #15%
    print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: 15 %".format(salario*1.15, salario*0.15))
elif salario <= 800: #12%
    print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: 12 %".format(salario*1.12, salario*0.12))
elif salario <= 1200: #10%
    print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: 10 %".format(salario*1.1, salario*0.1))
elif salario <= 2000: #7%
    print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: 7 %".format(salario*1.07, salario*0.07))
else: #4%
    print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: 4 %".format(salario*1.04, salario*0.04))

File: C#/Iniciante/1048 - Aumento de Salário.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 10:35:07
*/
using System;
using System.Globalization;

namespace URI {
    class URI {
        static void Main(string[] args)
        {
            double salario, ganhou;
            int reajuste;
            salario = double.Parse(Console.ReadLine());
            if (salario < 400.01)
            {
                reajuste = 15;
            }
            else if (salario < 800.01)
            {
                reajuste = 12;
            }
            else if(salario < 1200.01)
            {
                reajuste = 10;
            }
            else if (salario <= 2000)
            {
                reajuste = 7;
            }
            else
            {
                reajuste = 4;
            }

            ganhou = salario * reajuste / (double) 100;
            Console.WriteLine($"Novo salario: {(salario + ganhou).ToString("F2", CultureInfo.InvariantCulture)}");
            Console.WriteLine($"Reajuste ganho: {ganhou.ToString("F2", CultureInfo.InvariantCulture)}");
            Console.WriteLine($"Em percentual: {reajuste} %");
        }
    }
}
